Modules/Relationships/src/Relationships.Application/Identities/IdentityDeleter.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Backbone.BuildingBlocks.Application.Identities;
+using Backbone.DevelopmentKit.Identity.ValueObjects;
+using Backbone.Modules.Relationships.Application.Relationships.Commands.DeleteRelationshipsOfIdentity;
+using Backbone.Modules.Relationships.Application.RelationshipTemplates.Commands.DeleteRelationshipTemplatesOfIdentity;
+using MediatR;
+
+namespace Backbone.Modules.Relationships.Application.Identities;
+public class IdentityDeleter : IIdentityDeleter
+{
+    private readonly IMediator _mediator;
+
+    public IdentityDeleter(IMediator mediator)
+    {
+        _mediator = mediator;
+    }
+
+    public async Task Delete(IdentityAddress identityAddress)
+    {
+        await _mediator.Send(new DeleteRelationshipsOfIdentityCommand(identityAddress));
+        await _mediator.Send(new DeleteRelationshipTemplatesOfIdentityCommand(identityAddress));
+    }
```

Comment:
No, because an allocation is created for each peer template fetched by the identity to be deleted. 

But: this comment is kind of outdated anyway. As discussed in another channel, we shouldn't DELETE allocations, but instead replace the address of the identity to be deleted with a placeholder.